pygwalker/utils/gwalker_props.py

```diff
@@ -245,6 +251,12 @@ def getPropGetter(df: DataFrame) -> DataFramePropGetter:
         if isinstance(df, pd.DataFrame):
             __classname2method[pd.DataFrame] = buildPandasPropGetter()
             return __classname2method[pd.DataFrame]
+    
+    if 'modin' in sys.modules:
+        from modin import pandas as mpd
+        if isinstance(df, mpd.DataFrame):
+            __classname2method[mpd.DataFrame] = buildModinPropGetter()
```

Comment:
`__classname2method[mpd.DataFrame] = buildPandasPropGetter(mpd)`

Since the main branch has a new commit, you need to rebase the latest main branch.😂